packages/e2e/cypress/integration/ui/components/authenticator/useAuthenticator/useAuthenticator.ts

```diff
@@ -0,0 +1,7 @@
+import { Then, When } from 'cypress-cucumber-preprocessor/steps';
+
+Then('I see a valid greetings message', () => {
+  cy.findByRole('document')
+    .contains(new RegExp('^Hello, .+!$', 'i'))
```

Comment:
Checks if the message has "Hello, {username}" structure. Kept it generic to not expose the test username 😈